NEWS.md

```diff
@@ -4,3 +4,6 @@
 * Changed project structure to contain server and client parts. Client builds as a lightweight java library
 * Extended Event schema
 * Added samples
+* Applied Liquibase scripting tool to manage database tables
```

Comment:
From my perspective, the news should inform prospective adopters or clients of the meaningful changes. I don't really follow how these advise an adopter or client about what has changed.